conftest.py

```diff
@@ -122,3 +123,36 @@ def pytest_configure(config):
     #
     # Somehow the default database is always in memory, though.
     settings.DATABASES['transient']['TEST_NAME'] = ':memory:'
+
+
+@pytest.yield_fixture()
+def installed_packages_catalog():
+    with mock.patch('ideascube.serveradmin.catalog.Catalog', spec=True) as MockClass:
+        pseudo_package1 = mock.NonCallableMock(spec_set='id name description template_id language staff_only'.split())
+        pseudo_package1.configure_mock(id='test.package1',
+                                       name='Test package1',
+                                       description='Test package1 description',
+                                       template_id='template_name',
+                                       language='fr',
+                                       staff_only=False)
+        pseudo_package2 = mock.NonCallableMock(spec_set='id name description template_id language staff_only'.split())
+        pseudo_package2.configure_mock(id='test.package2',
+                                       name='Test package2',
+                                       description='Test package2 description',
+                                       template_id='template_name',
+                                       language='fr',
+                                       staff_only=False)
+        pseudo_package3 = mock.NonCallableMock(spec_set='id name description'.split())
+        pseudo_package3.configure_mock(id='test.package3',
+                                       name='Test package3 no template',
+                                       description='Test package3.\n'
+                                                    'A package without template'
+                                       )
+
+        instance = MockClass.return_value
+
+        instance.list_installed.side_effect = lambda ids_list: [
+            p for p in (pseudo_package1, pseudo_package2, pseudo_package3)
+            if ( (p.id in ids_list) or ('*' in ids_list) )]
+
+        yield MockClass
```

Comment:
Love the way you're mocking the Catalog here, I've had difficulties doing similar things in the past because the `mock` documentation lacks clear examples, and this is a great one!